skribenten-web/frontend/package-lock.json

```diff
@@ -645,9 +645,9 @@
       "integrity": "sha512-snKqtPW01tN0ui7yu9rGv69aJXr/a/Ywvl11sUjNtEcRc+ng/mQriFL0wLXMef74iHa/EkftbDzU9F8iFbH+zg=="
     },
```

Comment:
Er det meningen å nedgradere pakker i package-lock.json?

Reply:
hmm, nei. Resetter denne. skjedde sikkert da jeg endret cypress versjonen 3 ganger for å teste